src/popup/components/QrCode.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div id="qr-canvas" />
+</template>
+<script>
+import QRCodeStyling from '@solana/qr-code-styling';
+
+import SHLogo from '../../icons/logo-smal-blue.png';
+
+export default {
```

Comment:
To be honest this component begs to be written with the composition API and TS :)
Using TypeScript here would give us the benefit of type checking of the settings passed to the `QRCodeStyling`.

And this would result of finding that the `typeNumber: '0',` is wrong, because `typeNumber` accepts only numbers and undefined.

If you want help with that please ping me.